src/audio/aria/aria_generic.c

```diff
@@ -40,7 +40,7 @@ inline void aria_algo_calc_gain(struct aria_data *cd, size_t gain_idx,
 		src = audio_stream_wrap(source, src + n);
 		samples -= n;
 	}
-
+	max_data = max_data << 8;
```

Comment:
Doesn't this overflow with s32_le format?

Reply:
The concept of the fix has changed and this problem no longer exists.